master/internal/db/postgres_trial.go

```diff
@@ -685,6 +700,17 @@ func AddCheckpointMetadata(ctx context.Context, m *model.CheckpointV2, runID int
 	return nil
 }
 
+func experimentsProjectID(ctx context.Context, experimentID int, tx bun.IDB) (int, error) {
```

Comment:
maybe change the parameter order to the more standard one?

```suggestion
func experimentsProjectID(ctx context.Context, tx bun.IDB, experimentID int) (int, error) {
```